tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -124,9 +125,6 @@ describe(__filename, () => {
     it('waits for an addon and reviews to load', () => {
       const location = fakeRouterLocation();
       const root = render({ addon: null, location });
-      expect(root.find('.AddonReviewList-header-icon img').prop('src')).toEqual(
```

Comment:
since addon type is not known yet, this is hard to test unless i remove this like i've done here. Not sure if that's ideal tho... I am wondering if should revert this approach and just use styles to hide the icon and then we can leave this test? @willdurand , thoughts?

Reply:
Why do you want to remove it? It seems like you still want the `fallbackIcon`, no?

Reply:
Static themes don't have their own icons and the default icon isn't supposed to be displayed. The [issue here](https://github.com/mozilla/addons-frontend/issues/5277#issuecomment-400533331) mentions white space but since it stands alone (vs in a list with other types), I was going to remove it and left align the text

Reply:
But when there is no addon yet, it should display the fallback icon, no?

Once the addon is loaded, you can either remove it or replace it with the actual addon's icon, but before I would keep the fallback icon.

Reply:
You're right, it probably should keep the fallback when there is no addon. This failing test made me rethink my approach so I wanted a 2nd opinion. Cool tho, that helps me : ) So I'll add that back and will use css to hide the static theme icon. thank you!